geomstats/geometry/lie_group.py

```diff
@@ -368,19 +381,63 @@ def lie_bracket(
         if point_type is None:
```

Comment:
How come this code is not tested? I thought we had tests for the lie_bracket.

Reply:
I did fix a test for lie bracket in so3 (with cross product), so it overrides this code